integrations/database/mariadb/install.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# Copyright KubeArchive Authors
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit -o errtrace
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd ${SCRIPT_DIR}
+
+NAMESPACE="mariadb"
+
+# Install MariaDB operator.
+kubectl create ns mariadb-operator --dry-run=client -o yaml | kubectl apply -f -
+helm upgrade --install --namespace mariadb-operator mariadb-operator mariadb-operator/mariadb-operator
+kubectl -n mariadb-operator rollout status deployment --timeout=90s
+
+# Create the MariaDB database server.
+kubectl create ns ${NAMESPACE} --dry-run=client -o yaml | kubectl apply -f -
+kubectl -n ${NAMESPACE} apply -f .
+kubectl -n ${NAMESPACE} wait statefulset/kubearchive --for=create --timeout=60s
+kubectl -n ${NAMESPACE} wait pod --all --for=condition=ready --timeout=60s
+
+# Create the kubearchive database
+LOCAL_PORT=3307
+echo Forwarding local port ${LOCAL_PORT} to mariadb/kubearchive:3306.
+export ROOT_PWD=$(kubectl -n ${NAMESPACE} get secret kubearchive -o jsonpath='{.data.root-password}' | base64 --decode)
+kubectl -n ${NAMESPACE} port-forward service/kubearchive ${LOCAL_PORT}:3306 & # >& /dev/null &
+
+echo Waiting for ${LOCAL_PORT} to become available.
```

Comment:
I suggest adding a bit more text here. You should at least have the text  "port" here.

```suggestion
echo Waiting for port ${LOCAL_PORT} to become available.
```

Reply:
Fixed.